### ISSUE TITLE
mean_squared_error accuracy

### ISSUE THREAD

**Issue by firatsarlar** [tensorflow/tensorflow#54447]
My currenct observation with "mean_squared_error" loss,
If you use mean_squared_error loss and metrics= ["acc"] , 
during training you can't see any meaningfull accuracy,
I wrote my custom acc. function , it's very hard for me to make a PR, because of project complexty,

Here is my fn. 
```
def mean_squared_error_acc(y_true, y_pred):
    loss = tf.losses.mean_squared_error(y_true,y_pred)    
    return 100 * ( 1 - tf.math.sqrt(loss))

```
( by the way, I'm at very begining of my ML experience, and this issue would be meaningless , please lead me to right direction if I'm wrong) 

--Edit--
I use one of my colab experiments to reproduce issue
This is colab link
https://colab.research.google.com/drive/1hYG4RmPaUlnEcRm28NE6Y3HxIMoq1DTM?usp=sharing

These are the cells related the issue : 

https://colab.research.google.com/drive/1hYG4RmPaUlnEcRm28NE6Y3HxIMoq1DTM#scrollTo=lL3kJjI-cXLS&line=4&uniqifier=1

https://colab.research.google.com/drive/1hYG4RmPaUlnEcRm28NE6Y3HxIMoq1DTM#scrollTo=eYb8YEiRvVuK&line=3&uniqifier=1

**Comment by tilakrayal** [comment]
@firatsarlar,
Please post this issue on [keras-team/keras repo](https://github.com/keras-team/keras/issues).
To know more refer to:
https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999

**Comment by Cheril311** [comment]
@firatsarlar please update the issue on keras, we can discuss there

**Comment by tilakrayal** [comment]
@firatsarlar ,
Can you please post this issue on [keras-team/keras repo](https://github.com/keras-team/keras/issues) as this issue is more related to Keras.
